cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -225,7 +225,12 @@ function move_sriov_pfs_netns_to_node {
   mkdir -p /var/run/netns/
   ln -sf /proc/$pid/ns/net "/var/run/netns/$node"
 
-  local -r sriov_pfs=( /sys/class/net/*/device/sriov_numvfs )
+  local -r sriov_pfs=( $(find /sys/class/net/*/device/sriov_numvfs) )
+  if [ ${#sriov_pfs[@]} -eq 0 ]; then
+    echo "FATAL: Could not find available sriov PFs"
+    exit 1
```

Comment:
it wont fail, try it, there is local -r that cause it to not fail.

Reply:
So why using `find` at all if you masks the error? 
I mean there is no different then how it was before

Reply:
there is a difference.
The previous line would have an item even if there are no PF
and then to check if its empty, it will be awkward:
check if the first element is `/sys/class/net/*/device/sriov_numvfs` 
Please try and see it.